.github/PULL_REQUEST_TEMPLATE/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,50 @@
+# Pull Request Template
+
+## Description
+
+Please provide a brief description of the change and mention the issue it addresses.Also, include relevant motivation and context. List any dependencies that are required for this change.
+
+Please delete options that are not relevant and please make sure that you X the appropriate boxes.
+
+Fixes # (issue)
+
```

Comment:
[suggestion] Consider giving a different heading and instructions on how to link the issue. This would help contributors understand how to do this process. 
```suggestion
## Related Tickets and Documents 
<!-- Please use the following format(s) to link your issue(s):
Closes # issue number (e.g, 123)

Relates to # issue number (e.g, 123)
-->
```